src/components/dashboard/TeamMembersTable/TeamEditInfo.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { useMutation } from '@apollo/client';
+import { useApolloClient } from '@apollo/client';
+import { teamInfo } from 'data/actions/auth';
+import { UPDATE_TEAM_INFO } from '../../../data/gql/team';
+import Toast from 'components/dashboard/Toast/Toast';
+
+import { Box, Text, VStack, Input, Heading, useToast, Button } from '@chakra-ui/react';
+
+const EditTeamInfo = ({ setOnEditPage, teamId, teamName, teamAffiliation }) => {
+  const client = useApolloClient();
+  const [newTeamName, setNewTeamName] = useState(teamName);
+  const [newTeamAffiliation, setNewTeamAffiliation] = useState(teamAffiliation);
+  const [editTeam, { loading: loadingTeamChange, error: teamChangeError }] = useMutation(UPDATE_TEAM_INFO);
+  useEffect(() => {});
+  const dispatch = useDispatch();
+  const toast = useToast();
+
+  const onChangeTeamName = e => {
+    if (e.target.value === '') {
+      setNewTeamName(teamName);
```

Comment:
are we sure we want this behaviour? doesn't seem ideal

Reply:
Yeah, I wasn't sure what the behaviour should be if a user leaves a field empty so I did this. But if we're requiring the team name to be set each time, if a user just wants to update the team affiliation they're gonna have to manually write in their team name?

Reply:
it'll just retain the previous value, wouldn't it?

Reply:
Oh sorry I didn't explain my full train of thought. If a user selects the box and accidentally types in something then deletes what they typed, then it'd send an empty string in the query.

Reply:
that's true! see my comment on adding `isRequired` prop to `<Input>`